src/stream/sosemanuk/sosemanuk.c

```diff
@@ -715,13 +585,8 @@ static ulong32 mul_ia[] = {
 /*
  * Compute the next block of bits of output stream. This is equivalent
  * to one full rotation of the shift register.
- *
- * If SOSEMANUK_SPEED is defined, this function takes an extra parameter
- * "counter". The function then returns the sum of all produced
- * 32-bit words, in an "ulong32". That sum prevents the compiler from
- * optimizing out part of the computation.
  */
-static INLINE void sosemanuk_internal(sosemanuk_run_context *rc)
+static LTC_INLINE void sosemanuk_internal(sosemanuk_state *ss)
```

Comment:
This should be called `_sosemanuk_internal` ( @karel-m it looks like `helper.pl` doesn't detect that because of the additional keyword)

Reply:
I have fixed helper.pl (in develop)